jira/issue.go

```diff
@@ -69,3 +69,33 @@ func CreateIssue(args *CreateIssueArgs) (string, error) {
 	}
 	return issue.Key, nil
 }
+
+func GetActiveSprint(commonArgs *CommonArgs) (*gojira.Sprint, error) {
+	client, err := GetClient(commonArgs)
+	if err != nil {
+		return nil, fmt.Errorf("get client: %w", err)
+	}
+
+	sprints, _, err := client.Board.GetAllSprintsWithOptions(commonArgs.Board, &gojira.GetAllSprintsOptions{
+		State: "active",
+	})
+	if err != nil {
+		return nil, err
+	}
+	for _, sprint := range sprints.Values {
+		if sprint.State == "active" {
```

Comment:
Boards technically can have multiple active sprints. The sucky bit is if an issue is tracked on two different sprints you can get two results here.